data/_ui/tooltips.txt

```diff
@@ -343,25 +346,25 @@ tip "scramble resistance heat:"
 	`Heat generated per second when resisting scrambling damage.`
 
 tip "jump speed:"
-	`How slow the ship must be moving to make a jump.`
+	`The maximum velocity the ship can jump at.`
 
 tip "jump fuel:"
-	`How much fuel is consumed when you jump between systems using this drive.`
+	`Fuel consumed when jumping between systems using this drive.`
 
 tip "jump range:"
-	`How far you can jump between systems using this drive.`
+	`The maximum distance this drive can jump.`
 
 tip "jump mass cost:"
-	`How much fuel this drive consumes, per 100 tons of mass. Additive with jump fuel.`
+	`Fuel consumed per 100 tons of mass when jumping. Additive with jump fuel.`
 
 tip "jump base mass:"
-	`A modifier to the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`
+	`Modifies the impact your ship's mass has on the jump mass cost. Ships below this mass will have a cheaper jump cost than the base jump fuel cost, while ships above this mass will have more expensive jumps.`
 
 tip "leak protection:"
-	`Protection against weapons that cause fuel leakage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
+	`Protection provided against fuel leakage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
```

Comment:
Should this be changed to "Protection provided against leak damage"?

Reply:
```suggestion
	`Protection provided against leak damage. If you add more than one outfit with this attribute, each additional one is less effective: a total value of 1 results in a 1 percent reduction in leak damage, while a total value of 11 only results in a 10 percent reduction.`
```